src/js/pyodide.js

```diff
@@ -235,11 +235,13 @@ def temp(Module):
   Module.pyodide_py = pyodide
 `);
 
-  Module.saveState = () => Module.pyodide_py._state.save_state();
-  Module.restoreState = (state) =>
-    Module.pyodide_py._state.restore_state(state);
-
   Module.init_dict.get("temp")(Module);
+
+  let _state = Module.pyodide_py._state;
+  let save_state = _state.save_state;
+  let restore_state = _state.restore_state;
+  Module.saveState = () => save_state();
+  Module.restoreState = (state) => restore_state(state);
```

Comment:
We now have those in api.js I think? Syncing with main would probably show conflicts here.

Reply:
Yeah they are better in a different file.